drupal_module_template..travis.yml

```diff
@@ -0,0 +1,78 @@
+language: bash
+
+sudo: required
+
+services:
+  - docker
+
+addons:
+  apt:
+    packages:
+      - docker-ce
+
+cache:
+  directories:
+    - $HOME/.composer/cache/files
+
+notifications:
+  email:
+    on_success: never
+    on_failure: always
+
+env:
+  global:
+    - DOCKER_COMPOSE_VERSION=1.24.0
+    - THREADS=4
+    - COMPOSER_GLOBAL_OPTIONS="--no-interaction --no-suggest -o"
+  matrix:
+    - CHECK_CS=true
+    - RUN_TESTS=true
+    - RUN_TESTS=true DEPENDENCIES="--prefer-lowest"
+    - RUN_TESTS=true DB_DRIVER=pgsql DB_IMAGE=wodby/postgres:9.6-1.3.1
+    - RUN_TESTS=true PHP_IMAGE=wodby/drupal-php:7.2
+    # TODO Restore ability to run tests with latest Drupal 8.x dev.
+
+matrix:
+  fast_finish: true
+
+before_install:
+  - |
+    sudo rm /usr/local/bin/docker-compose
+    curl -L https://github.com/docker/compose/releases/download/${DOCKER_COMPOSE_VERSION}/docker-compose-`uname -s`-`uname -m` > docker-compose
+    chmod +x docker-compose && sudo mv docker-compose /usr/local/bin
+  # Dummy permission fix for Travis because the default user's id is not 1000 (which is also the id of the "wodby" user
+  # in the containers).
+  # uid=2000(travis) gid=2000(travis) groups=2000(travis),999(docker)
+  - sudo chown -R 1000:travis . $HOME/.composer
```

Comment:
Nope, we can not use `$(id -u)` because that would return 2000 as you can see in the comment above. But all files created within the container is owned by the wodby user - also PHP-fpm is running as that user - which user id is the Unix default, 1000. 

Somewhen they change the Travis user's id to 2000 and this is the problem.
Related: https://github.com/SciSpike/yaktor-issues/issues/76

Reply:
So the comment and your additional comment didn't help me getting out of the woods – can we reiterate on this comment? so a newcomer like me would know what that `1000` magic number is, and where is it coming from?

Reply:
Updated the comment, I hope the new comment helps to understand the problem.